components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -23,40 +27,101 @@ class SchoolCard extends Component {
     logoSource: PropTypes.string.isRequired,
     name: PropTypes.string.isRequired,
     website: PropTypes.string.isRequired,
+    toggleModal: PropTypes.func.isRequired,
   };
 
   static defaultProps = {
-    hasHousing: false,
+    hasHousing: true,
   };
 
-  state = {
-    isFrontOfCardShowing: true,
+  toggleModalClick = () => {
+    const { name, locations, toggleModal } = this.props;
+    toggleModal({ name, locations });
   };
 
-  showBackOfCard = () => this.setState({ isFrontOfCardShowing: false });
+  render() {
+    const { props } = this;
 
-  showFrontOfCard = () => this.setState({ isFrontOfCardShowing: true });
+    const hasManyLocations = props.locations.length > 1;
 
-  render() {
-    const { props, state } = this;
+    let locationText = 'Multiple locations';
```

Comment:
You could extract this logic to a method at the top of the folder. Maybe, `getSchoolLocationText`.

One benefit is you can easily add test coverage for it.